PennMobile/Home/HomeViewModel.swift

```diff
@@ -24,6 +24,8 @@ extension Optional {
 @MainActor class StandardHomeViewModel: HomeViewModel {
```

Comment:
tbh I think a general refactor of `HomeViewModel` is overdue, but we can punt that to later